recipes/opensubdiv/all/conanfile.py

```diff
@@ -0,0 +1,137 @@
+import functools
+import os
+
+from conan.tools.microsoft import is_msvc
+from conans import ConanFile, CMake, tools
+
+
+class OpenSubdivConan(ConanFile):
+    name = "opensubdiv"
+    license = "Modified Apache 2.0 License"
```

Comment:
```suggestion
    license = "Modified-Apache-2.0"
```
Avoid empty space due the search engine. Try to use short names

Reply:
fixed